src/Makefile.am

```diff
@@ -240,6 +240,7 @@ BITCOIN_CORE_H = \
   util/memory.h \
   util/message.h \
   util/moneystr.h \
+  util/tokenpipe.h \
```

Comment:
nit, sort

Reply:
Strange, I guess I had this module named differently at first :smile: